src/bench_ecmult.c

```diff
@@ -123,6 +124,31 @@ static void bench_ecmult_const_teardown(void* arg, int iters) {
     bench_ecmult_teardown_helper(data, &data->offset1, &data->offset2, NULL, iters);
 }
 
+static void bench_ecmult_const_xonly(void* arg, int iters) {
+    bench_data* data = (bench_data*)arg;
+    int i;
+
+    for (i = 0; i < iters; ++i) {
+        const secp256k1_ge* pubkey = &data->pubkeys[(data->offset1+i) % POINTS];
+        const secp256k1_scalar* scalar = &data->scalars[(data->offset2+i) % POINTS];
+        secp256k1_ecmult_const_xonly(&data->output_xonly[i], &pubkey->x, NULL, scalar, 1);
+    }
```

Comment:
nit: the argument is critical to performance and this documents it a bit better
```suggestion
    for (i = 0; i < iters; ++i) {
        const secp256k1_ge* pubkey = &data->pubkeys[(data->offset1+i) % POINTS];
        const secp256k1_scalar* scalar = &data->scalars[(data->offset2+i) % POINTS];
        int known_on_curve = 1;
        secp256k1_ecmult_const_xonly(&data->output_xonly[i], &pubkey->x, NULL, scalar, known_on_curve);
    }
```